lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -11,7 +11,9 @@ class LighthouseSupplementalDocumentUploadProvider
   # Maps VA's internal Document Types to the correct document_type attribute for a Lighthouse526DocumentUpload polling
   # record. We need this to create a valid polling record
   POLLING_DOCUMENT_TYPES = {
-    'L023' => Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE
+    'L023' => Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE,
+    'L228' => Lighthouse526DocumentUpload::FORM_0781_DOCUMENT_TYPE,
+    'L229' => Lighthouse526DocumentUpload::FORM_0781A_DOCUMENT_TYPE
```

Comment:
Oops I actually ended up changing how this works, I had to do some trickery for the vet uploads, and I ended up removing this list in the pending veteran upload pr that is still in review.

If you could make the change to use [this method](https://github.com/department-of-veterans-affairs/vets-api/pull/19117/files#diff-a0a3ecd0f4e7798e8a50d8a0c7807733ceec98c04dec72c0fd9a4f64d333ef6dR182) instead that should take care of it.

Reply:
Disregard see below